cmd/update-main.go

```diff
@@ -68,90 +68,110 @@ const (
 	minioReleaseURL           = "https://dl.minio.io/server/minio/release/" + runtime.GOOS + "-" + runtime.GOARCH + "/"
 )
 
-func getCurrentReleaseTime(minioVersion, minioBinaryPath string) (releaseTime time.Time, err error) {
-	if releaseTime, err = time.Parse(time.RFC3339, minioVersion); err == nil {
+type updateParams struct {
+	versionStr         string
+	serverBinaryPath   string
+	cgroupFile         string
+	releaseChecksumURL string
+	serverMode         string
+}
```

Comment:
Added more comments - this set of parameters enables writing tests for various pieces of logic. It should be more readable now.